types/electron-devtools-installer/electron-devtools-installer-tests.ts

```diff
@@ -1,11 +1,4 @@
-import installExtension, {
-    EMBER_INSPECTOR, REACT_DEVELOPER_TOOLS,
-    BACKBONE_DEBUGGER, JQUERY_DEBUGGER,
-    ANGULARJS_BATARANG, VUEJS_DEVTOOLS,
-    REDUX_DEVTOOLS, REACT_PERF,
-    CYCLEJS_DEVTOOL, APOLLO_DEVELOPER_TOOLS,
-    MOBX_DEVTOOLS
-} from 'electron-devtools-installer';
+import installExtension, { ANGULARJS_BATARANG, APOLLO_DEVELOPER_TOOLS, BACKBONE_DEBUGGER, CYCLEJS_DEVTOOL, EMBER_INSPECTOR, JQUERY_DEBUGGER, MOBX_DEVTOOLS, REACT_DEVELOPER_TOOLS, REACT_PERF, REDUX_DEVTOOLS, VUEJS_DEVTOOLS } from 'electron-devtools-installer';
```

Comment:
You shouldn't change formatting.

Reply:
That was some trigger happy editor plugin, reverted it now.